caddyhttp/httpserver/plugin.go

```diff
@@ -410,6 +410,9 @@ var directives = []string{
 	"gzip",
 	"header",
 	"errors",
+	"filter", // github.com/echocat/caddy-filter - Required at this position because if it is
+	//                                             enabled it captures the response of following
+	//                                             middlewares and modifies it.
```

Comment:
We can probably lob off the explanation -- if we did that for every directive, we'd have a bit of an unwieldy list. :) Keep the package path though. (Sorry, I know this absolute ordering isn't a great system, but it's the only correct way I can think of doing it!)